### PR TITLE
Module 5 new

### PR DESCRIPTION
**Pull request by jaybee1971**
This should be everything done bar stretch task 5 (I still need to complete exercise 3 in full, so will come back to running tests when I have all my test done!).
I also refactored the entire folder structure to put the code in one place and not have it mixed in with config files. This caused some minor issues with the .env file, but I seemed to worked around it by having 2 (one for gunicorn and one for pure flask)... hints on how to change this welcomed :) 
I couldn't get the volume mount to work correctly until I refactored the code into it's own directory, not sure why the docker logs showed it mounted OK, but once in own folder everything seemed to work fine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +41,12 @@ Packages required:
...
 ```
 
-To update any missing dependencies:
+To update any missing or new dependencies:
 ```bash
-$ poetry add --dev <package_name>
+$ poetry add <package_name>
 ```
 
 ## Trello API
```
Comment (Jonesey13) [inline README.md:52]: I'd add a note saying that you need the right todo list names in this section of the `README.md` (else the application will fail)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+FROM base-image as todo-dev
+# copy app code, test code and run flask
+# COPY entrypoint-dev.sh ./*.py ./
+# COPY ./templates/ ./templates/
+# COPY ./tests/ ./tests/
+COPY entrypoint-dev.sh ./
+EXPOSE 5000
+RUN chmod +x ./entrypoint-dev.sh
+ENTRYPOINT ["sh", "entrypoint-dev.sh"]
```
Comment (Jonesey13) [inline Dockerfile:33]: FYI this will fail with people using docker on some windows machines (can you see why?)

File: todo_app/trello_api.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from flask import session, current_app as app
```
Comment (Jonesey13) [inline todo_app/trello_api.py:1]: Not sure why we have so many code changes in this review?

**Review by Jonesey13 — APPROVED** [review]
This looks to be all working to me! The README documentation is also pretty good.

Took me quite a while to get this working (due to a couple of issues not pointed out by the README.md) so left a couple of comments below.

File: todo_app/trello_api.py (hunk 3 above)
Comment (jaybee1971) [reply todo_app/trello_api.py:1]: I cleaned up the project structure and moved all the code into it's own folder away from the config type stuff and docker files... this meant I had to change some class paths as it broke some of the import statements.

**Pull request merged**